BlackMarket/Views/ProductItem.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ProductItem.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 22/09/2023.
+//
+
+import SwiftUI
+
+
+struct ProductItem: View {
+  
+  var product: Product
+  
+  var body: some View {
+    VStack(alignment: .center) {
+      Image("blackmarket_chair")
```

Comment:
Generally speaking, please check out comments on previous PRs, in this case specifically to the image assets names